lib/specific_todo.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+class SpecificTodo extends StatelessWidget {
+  final String todo;
+  final bool isDone;
+  final int index;
+
+  const SpecificTodo(
```

Comment:
In flutter style, the constructors should appear in a class before anything else.